src/components/FilterTodos.tsx

```diff
@@ -0,0 +1,20 @@
+import { useMemo } from 'react';
+import { Status, OrderItems } from '../types/Type';
+
+export function FilterTodos(
+  orderItems: OrderItems[],
+  status: Status,
+): OrderItems[] {
+  const preparedTodos = useMemo(() => {
```

Comment:
Why does your helper function looks like a react component and `preparedTodos` wrapped with `useMemo` 🤯